.github/workflows/ee-nlc-snapshot-push.yml

```diff
@@ -15,6 +15,10 @@ env:
 jobs:
   push:
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        jdk: [ '11', '17', '21' ]
```

Comment:
Do we have approve from PM (or similar person) to add 3 different docker images?

Reply:
From @avtarraikmo - 

For now - lets support 11, 17 and 21 images.